trading-cards/contracts/evm/solidity/PaimaL2Contract.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
```

Comment:
This file should be removed before merging this. I copy-pasted this here because `@paima/evm-contracts` isn't on NPM yet